source/Depreciation.cpp

```diff
@@ -334,11 +334,11 @@ double Depreciation::Depreciate(int age) const
 {
 	if(age <= GRACE_PERIOD)
 		return 1.;
-	if(age >= MAX_AGE + GRACE_PERIOD)
```

Comment:
Was there a reason we didn't reduce `age` by `GRACE_PERIOD`, check if it was <= 0, and exit if so? Could we have then used the original calculations?

Reply:
I don't believe so, as the true age of fully depreciated outfits and ships would still be set to 1000, meaning that the depreciation calculation would use 1000 - 7 = 993 days, which isn't max depreciation.